### PR TITLE
[RFR] Added “IIFE” entry

### PR DESCRIPTION
**Pull request by rbrtsmith**
Add in description.

### CODE REVIEW

File: glossary/IIFE.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
-# IIFE
+IIFE Immediately Invoked Function Expression is a function that gets called immediately after declaration.  Is most often used to create a scoping context.
```
Comment (KittyGiraudel) [inline glossary/IIFE.md]: Leave the title up there please. :)
Also maybe _Immediately Invoked Function Expression_ could be in parentheses?

File: glossary/IIFE.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
 
-IIFE section to be completed.
+An IIFE be written with the calling brackets inside of the wrapping brackets:
+```javascript
+(function foo() {
+  // [body]
+}());
+````
```
Comment (KittyGiraudel) [inline glossary/IIFE.md]: One too many backtick here. ;)

File: glossary/IIFE.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+
+Or with the calling brackets on the outside
+```javascript
+(function foo() {
+  // [body]
+})();
+````
```
Comment (KittyGiraudel) [inline glossary/IIFE.md]: One too many backtick there. ;)

Comment (rbrtsmith) [inline glossary/IIFE.md]: should all be fixed now :)

On 8 December 2015 at 00:23, Hugo Giraudel notifications@github.com wrote:

> In glossary/IIFE.md
> https://github.com/HugoGiraudel/SJSJ/pull/62#discussion_r46894227:
> 
> > -IIFE section to be completed.
> > +An IIFE be written with the calling brackets inside of the wrapping brackets:
> > +```javascript
> > +(function foo() {
> > -  // [body]
> >   +}());
> >   +``
> >   +
> >   +Or with the calling brackets on the outside
> >   +`javascript
> >   +(function foo() {
> > -  // [body]
> >   +})();
> >   +````
> 
> One too many backtick there.; )
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HugoGiraudel/SJSJ/pull/62/files#r46894227.

## 

Kind regards,

_\- Robert Smith_
_Web Developer_

---

_W_: http://rbrtsmith.com/
_T_: +46(0)73 687 5773
_E_: rbrtsmith@googlemail.com

**Comment by KittyGiraudel** [comment]
Thanks!

**Pull request merged**